packages/openapi-v2/README.md

```diff
@@ -0,0 +1,77 @@
+@loopback/openapi-v2
+
+This package contains:
+
+- Decorators that describe LoopBack artifacts as metadata.
+- Uilities that tranfer LoopBack remoting metadata to swagger specifications. 
```

Comment:
Utilities that inspect controller classes to build OpenAPI v2 (Swagger) specifications from REST api mapping metadata.